### PR TITLE
Lso

### PR DESCRIPTION
**Pull request by brittcyr**

### CODE REVIEW

File: utils/instructions/Dual/index.ts [hunk 1]
```diff
@@ -404,3 +432,227 @@ export async function getWithdrawInstruction({
...
+    if (!baseMint) {
+      return {
+        serializedInstruction,
+        isValid: false,
+        governance: form.baseTreasury?.governance,
+        additionalSerializedInstructions: [],
+      }
+    }
+
+    for (const strike of form.strikes.split(',')) {
```
Comment (DonDuala) [inline utils/instructions/Dual/index.ts:614]: should this be within the check above for a valid SO?

File: utils/instructions/Dual/index.ts [hunk 2]
```diff
@@ -404,3 +432,227 @@ export async function getWithdrawInstruction({
...
+      form.quoteTreasury.pubkey,
+      form.payer.extensions.transferAddress!,
+      issueAuthority,
+    )
+
+    additionalSerializedInstructions.push(
+      serializeInstructionToBase64(configInstruction)
+    )
+
+    //after everything we close helper account
```
Comment (DonDuala) [inline utils/instructions/Dual/index.ts:546]: Is this new compared to the update to Staking Options from https://github.com/solana-labs/governance-ui/commit/dafd05dfa77a5145110a6a89579983919db7ba94 ?

File: utils/instructions/Dual/index.ts (hunk 1 above)
Comment (brittcyr) [reply utils/instructions/Dual/index.ts:614]: We could get thorough with the validations, but ultimately, this all will need to be redone anyways. The Staking Options instruction in realms has always been a hack job for the narrow use case

File: utils/instructions/Dual/index.ts (hunk 2 above)
Comment (brittcyr) [reply utils/instructions/Dual/index.ts:546]: This is a more complete version of the poorly named Staking Options instruction in realms currently. This is similar except it doesnt do the follow up initStrike and it also uses a separate issue authority

Comment (DonDuala) [reply utils/instructions/Dual/index.ts:546]: Specifically referring to the closure of the helper account. That wasn't done in Staking Options gov UI and had to have a separate proposal for closing old accounts, correct? If so maybe worth adding there too with this PR to simplify it for mango

Comment (brittcyr) [reply utils/instructions/Dual/index.ts:546]: Are you saying the current StakingOptions governance ui instruction isnt working?
As far as I am aware, this still works.
https://github.com/Dual-Finance/governance-ui/blob/133fa4c0af844ecadddb76f04576ee75980765d6/utils/instructions/Dual/index.ts#L226

Comment (DonDuala) [reply utils/instructions/Dual/index.ts:546]: You're right. I had missed this, thought the edits were only to withdraw fct but it's also included in config

**Comment by DonDuala** [comment]
Close this?

**Comment by brittcyr** [comment]
> Close this?

yes

**Pull request closed without merging**